src/Symfony/Component/Dsn/README.md

```diff
@@ -0,0 +1,13 @@
+Dsn Component
+=============
+
+Provides utilities for handle Data Source Names.
```

Comment:
for creating connections using DSN (Data Source Names) strings.

Reply:
fixed